modules/study/src/main/ChapterRepo.scala

```diff
@@ -127,6 +127,14 @@ final class ChapterRepo(val coll: AsyncColl)(using Executor, akka.stream.Materia
 
   def setName(id: StudyChapterId, name: StudyChapterName) = coll(_.updateField($id(id), "name", name)).void
 
+  // used only by the relay module for the follow fide players feature
+  def hasNotified(id: StudyChapterId): Fu[Boolean] =
+    coll(_.exists($doc($id(id), "notified" -> id)))
+
+  // used only by the relay module for the follow fide players feature
+  def setNotified(id: StudyChapterId): Funit =
+    coll(_.update.one($id(id), $addToSet("notified" -> id)).void)
```

Comment:
it's the same `id` that selects the chapter, and is added to its `notified` set. It doesn't seem useful to have a chapter doc like `{_id: 'abcdefgh', notified: ['abcdefgh']}`